scripts/particle_rugosity/ReadMe.md

```diff
@@ -0,0 +1,85 @@
+# Particle Rugosity
+
+## Summary
+Simulates the particle shape of the given crystal structure using BFDH mophology 
+prediction, then calculates the particle rugosity by determining the rugosity 
+of the facets and calculating the weighted average (by facet area).
+
+### Relevance
+Rugosity is considered to be a factor in nucleation, with rougher crystal surfaces 
+having higher energies and being generally less stable. This property has been used
+in rationalizing the relative stability or probability of forming one polymorph 
+(or putative crystal structure from Crystal Structure Prediction, CSP) over another.
+
+Using a protocol similar to that reported by [Cruz-Cabeza and colleagues](https://doi.org/10.1002/anie.202006939) this script
+takes a crystal structure or refcode as an argument and will use the CCDC BFDH 
+morphology module to simulate the predicted particle shape. The facet hkl planes are
+then obtained and the rugosity of these planes is calculated using the [CCDC rugosity function](https://downloads.ccdc.cam.ac.uk/documentation/API/descriptive_docs/particle.html) (including a scan of 
+different off-sets from the origin to identify the minimum rugosity of that crystal 
+plane). Finally, the total particle rugosity is calculated by weighted average 
+according to the facet surface areas.
+
+## Requirements
+
+- Tested with CSD Python API version 3.7.9 on Linux and Windows
+- ccdc.particle
+- ccdc.morphology
+
+## Licensing Requirements
+
+- CSD-Core
+
+## Instructions on Running
+### Linux command line
+- load the CSD Python API Miniconda environment
+- call Python to read the script, add the crystal structure refcode or filename (local file)
+~~~
+$ python particle_rugosity.py AABHTZ
+The crystal structure AABHTZ has been loaded
+Calculating particle rugosity ...
+AABHTZ particle rugosity:  1.5073781143188552
+~~~
+OR
+~~~
+$ python particle_rugosity.py AABHTZ.cif
+~~~
+- results are printed and can be redirected to be saved in an output file
+~~~
+$ python particle_rugosity.py AABHTZ > AABHTZ_rugosity.out
+~~~
+
+### Windows CSD Python API
+- move the particle\_rugostiy.py file to the CCDC\Python\_API\_(year)\ folder 
+- to determine where this is on your computer, start the CSD Python API and enter the following commands
+~~~
+>>> import os
+>>> wd = os.getcwd()
+>>> print(wd)
+C:\Program Files\CCDC\Python_API_2022\
+~~~
+the last line is the folder location where you need to move the particle\_rugostiy.py file
+- now the script can be run in the CSD Python API to calculate the rugosity of any CSD entry using:
+~~~
+>>> import sys
+>>> import subprocess
+>>> subprocess.call([sys.executable, 'particle_rugosity.py', 'AABHTZ'])
+The crystal structure AABHTZ has been loaded
+Calculating particle rugosity ...
+AABHTZ particle rugosity:  1.5073781143188552
```

Comment:
how come you're using the subprocess call here? 
Could you not just import the script via the normal way of given the instruction is to put the script where the API is ?  
Such as 

```python 
from particle_rugosity import particle_rugosity 

particle_rugosity('AABHTZ')
``` 
Would that not work?

Reply:
fixed this